modules/couchbase/src/main/java/org/testcontainers/couchbase/AbstractCouchbaseWaitStrategy.java

```diff
@@ -0,0 +1,16 @@
+package org.testcontainers.couchbase;
+
+import org.slf4j.Logger;
+import org.testcontainers.containers.wait.strategy.AbstractWaitStrategy;
+
+abstract class AbstractCouchbaseWaitStrategy extends AbstractWaitStrategy {
+
+    CouchbaseContainer getContainer() {
+        return (CouchbaseContainer) waitStrategyTarget;
```

Comment:
It doesn't have to be an instance of `CouchbaseContainer` (checked the usages), the same strategy could be used for Docker Compose

Reply:
Yes, I'm aware of that, but I didn't find a better solution, I need the container in implemented classes, which was available in the previous implementation of ```AbstractWaitStrategy```. And in my case it is always a ```CouchbaseContainer```.

Reply:
Why would you need a ref to `CouchbaseContainer`?

Reply:
To get the container ip address and logger. otherwise I need to add them in constructor.

I'll check this tonight because pushing to github is blocked at work.

Reply:
you get `containerIp` from `WaitStrategyTarget` interface. 
Logger - I would rather prefer to create it inside the strategy, i.e. with `@Slf4j` annotation

Reply:
Ok, will do ASAP. Thanks!

Reply:
👍